### ISSUE TITLE
DNF error occured: DepsolveError 

### ISSUE THREAD

**Issue by zhangdexin** [osbuild/osbuild-composer#1394]
At centos8 stream, Is some dnf config error?
I typed blueprint file:
```
name = "base-bash"
description = "A base system with bash"
version = "0.0.6"
modules = []

[[packages]]
name = "bash"
version = "*"

[[groups]]
name = "anaconda-tools"

[customizations]

[[customizations.user]]
name = "root"
password = "123456"
groups = ["root"]

[[customizations.group]]
name = "root"
```

And I exec cmd with :
`composer-cli compose start base-bash qcow2`

Some error:
```
DNF error occured: DepsolveError: There was a problem depsolving ['@core', 'authselect-compat', 'chrony', 'cloud-init', 'cloud-utils-growpart', 'cockpit-system', 'cockpit-ws', 'dhcp-client', 'dnf', 'dnf-utils', 'dosfstools', 'dracut-norescue', 'shim-x64', 'NetworkManager', 'nfs-utils', 'oddjob', 'oddjob-mkhomedir', 'psmisc', 'python3-jsonschema', 'qemu-guest-agent', 'redhat-release', 'redhat-release-eula', 'rsync', 'subscription-manager-cockpit', 'tar', 'tcpdump', 'yum', 'bash', '@anaconda-tools', 'kernel', 'dracut-config-generic', 'grub2-pc', 'grub2-efi-x64']: 
 问题: package kdump-anaconda-addon-003-6.20210204git43c39c1.el8.noarch requires anaconda >= 32.18, but none of the providers can be installed
  - package anaconda-33.16.4.15-1.el8.0.2.x86_64 requires anaconda-install-env-deps = 33.16.4.15-1.el8.0.2, but none of the providers can be installed
  - package anaconda-33.16.4.15-1.el8.x86_64 requires anaconda-install-env-deps = 33.16.4.15-1.el8, but none of the providers can be installed
  - package anaconda-install-env-deps-33.16.4.15-1.el8.0.2.x86_64 requires udisks2-iscsi, but none of the providers can be installed
  - package anaconda-install-env-deps-33.16.4.15-1.el8.x86_64 requires udisks2-iscsi, but none of the providers can be installed
  - package udisks2-iscsi-2.9.0-6.el8.x86_64 requires udisks2(x86-64) = 2.9.0-6.el8, but none of the providers can be installed
  - package udisks2-iscsi-2.9.0-7.el8.x86_64 requires udisks2(x86-64) = 2.9.0-7.el8, but none of the providers can be installed
  - conflicting requests
  - package udisks2-2.9.0-6.el8.x86_64 is filtered out by exclude filtering
  - package udisks2-2.9.0-7.el8.x86_64 is filtered out by exclude filtering
```

_msehnout: edited markdown formatting for easier reading_

**Comment by msehnout** [comment]
This is a duplicate of: https://github.com/osbuild/osbuild-composer/issues/921

There is currently no workaround in CentOS8 stream, but once the patch from above mentioned issue lands in downstream, it will be possible to explicitly include `udisks2` to fix this issue.

**Comment by msehnout** [comment]
I'm closing this issue as it has already been fixed in upstream. The issue for CentOS8 stream should be reported in Red Hat bugzilla: https://bugzilla.redhat.com/

**Comment by msehnout** [comment]
The solution only works with latest unreleased upstream version of osbuild-composer. It will take some time before it is available in downstream repository such as the one in CentOS Stream or RHEL.

I cannot provide any timeframe on when this version is available in downstream distributions, but you can report a bug in the issue tracker of your distribution.
